release notes/v0.30.0.md

```diff
@@ -0,0 +1,123 @@
+k6 v0.30.0 is here! :tada: It was a bit of a slow after-holiday release, but it still packs a few major new features and improvements that users have been requesting for a long time!
+
+## New features
+
+### Shared memory read-only arrays between VUs ([#1739](https://github.com/loadimpact/k6/pull/1739))
+
+k6 has long had an issue with the handling of big data files with test fixtures. For example, if you have a huge `users.json` file with test users for your application:
+```js
+[
+  {"username": "user1", "password": "password1", "other": "some-long-data-....-1"},
+  {"username": "user2", "password": "password2", "other": "some-long-data-....-2"},
+  // ... ~1 million more users or more... :D
+  {"username": "user999999", "password": "password999999", "other": "some-long-data-....-999999"}
+]
+```
+
+If you just use `JSON.parse(open('users.json'))` in your script, then every VU will have a copy of the whole huge data set. Every VU in k6 is a separate JavaScript runtime, so there wasn't a thread-safe way to share data between them. Until now, that is!
+
+We've added a new built-in `SharedArray` object in the new `k6/data` module that allows VUs to share read-only data:
+```js
+import { SharedArray } from 'k6/data';
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+let users = new SharedArray('someName', function () {
+    // This function will be called only once, in the first init context
+    // execution. Every other VU will just get a memory-safe read-only reference
+    // to the already opened data.
```

Comment:
```suggestion
    // to the already returned data.
```

Reply:
not sure your suggestion makes it clearer, how about "already loaded data"?

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1809/commits/9b61b1bce833dfbbd61f2606860ca7f9831f6c42?